configure.ac

```diff
@@ -6790,6 +6790,7 @@ if test "$ENABLED_SP_NO_MALLOC" = "yes"; then
     AM_CCASFLAGS="$AM_CCASFLAGS -DWOLFSSL_SP_NO_MALLOC"
 fi
 if test "$ENABLED_SP_NONBLOCK" = "yes"; then
+    ENABLED_SP=yes
```

Comment:
Does nothing.
What will work is adding a check for ENABLES_SP being just "nonblack" and add ",yes" to it, before the parsing ENABLED_SP.

Reply:
Thank you so much! Will update as soon as possible.